features/step_definitions/menu_steps.rb

```diff
@@ -0,0 +1,8 @@
+Given(/^I have ordered something on the menu$/) do
+  self.order = menu.order('Spaghetti')
+end
+
+Then(/^my order should be placed$/) do
+  expect(order.class).to eq Order
```

Comment:
I wouldn't do a test like this, we tend not to worry about classes in ruby, you would maybe just check it's not nil